paper-tab.html

```diff
@@ -136,7 +137,33 @@
 
       _updateNoink: function() {
         this.noink = !!this.noink || !!this._parentNoink;
+      },
+
+      /**
+       * Transfer a keyboard click (enter/space) to the anchor tag contained
+       * within the tab
+       */
+      _onClick: function(event) {
+        if (this.hasAttribute('link')) {
+          var link;
+          var links = this.getEffectiveChildren('a');
```

Comment:
This should be `queryEffectiveChildren`; given that the tests are currently passing, I'd say there needs to be a couple more where the anchor isn't the first child of the `paper-tab` and where it's a descendant, but not a child.